### ISSUE TITLE
Just see a blank screen when I open it after downloaded Joplin

### ISSUE THREAD

**Issue by iLucis** [laurent22/joplin#8622]
**Device：** MacBook Pro, Apple M1 Max, 32GB

**System:** MacOS Sonoma 14 beta (23A5301h)

**Joplin:** Version 2.11.11

Is Joplin not compatible with MacOS beta systems? 
or Does it require a specific run environment? 
I downloaded Joplin V2.11.11 on my MacOS Sonoma 14, 
**but when I open it, I only see a blank screen, and I can't quit it normally, I have to force quit it.**

<img width="1210" alt="Snipaste_2023-08-06_16-15-10" src="https://github.com/laurent22/joplin/assets/132722267/fca05e04-b854-4b42-aa70-a28b2fcee0e1">

**Comment by laurent22** [comment]
What's the error in the console?

> If the application starts with a white screen, open **Help &gt; Toggle Development Tools** or **View &gt; Toggle Development Tools** in the menu. Then check in the console if there is any error or warning and please let us know about it.

**Comment by iLucis** [comment]
> What's the error in the console?
> 
> > If the application starts with a white screen, open **Help > Toggle Development Tools** or **View > Toggle Development Tools** in the menu. Then check in the console if there is any error or warning and please let us know about it.


Last login: Sun Aug  6 20:32:17 on console
/Applications/Joplin.app/Contents/MacOS/Joplin ; exit;
@MacBook-Pro ~ % /Applications/Joplin.app/Contents/MacOS/Joplin ; exit;
2023-08-06 22:48:46.703 Joplin[3717:109350] WARNING: Secure coding is not enabled for restorable state! Enable secure coding by implementing NSApplicationDelegate.applicationSupportsSecureRestorableState: and returning YES.
objc[3721]: Class WebSwapCGLLayer is implemented in both /System/Library/Frameworks/WebKit.framework/Versions/A/Frameworks/WebCore.framework/Versions/A/Frameworks/libANGLE-shared.dylib (0x7ffb43c3c070) and /Applications/Joplin.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Libraries/libGLESv2.dylib (0x11092ece0). One of the two will be used. Which one is undefined.
(node:3724) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 pluginMessage listeners added to [EventEmitter]. Use emitter.setMaxListeners() to increase limit
(Use `Joplin Helper (Renderer) --trace-warnings ...` to show where the warning was created)

<img width="585" alt="Snipaste_2023-08-06_23-06-42" src="https://github.com/laurent22/joplin/assets/132722267/89925a8b-969e-4c59-b1c4-34447a9bedf5">

**Comment by laurent22** [comment]
What's the error in the console?

> If the application starts with a white screen, open **Help &gt; Toggle Development Tools** or **View &gt; Toggle Development Tools** in the menu. Then check in the console if there is any error or warning and please let us know about it.

**Comment by yodahome** [comment]
I am not sure whether this is the right issue to add this info to, but just in case anybody has the same problem:

I noticed a similar behavior when I first tried upgrading version 2.11.11 on my Mac Mini M1 and Macbook Pro M1 Pro to the arm64 version. I did open the Development Tools, but the console was empty, no errors. I then changed back to the intel version, which worked. Just now I tried again with version 2.12.10 with the same result. 
I then decided to successively deactivate extensions I had installed (had to switch back to intel first) and found the culprit to be the macOS theme v1.3.13 which apparently kept Joplin arm64 from starting up. It works when I deactivate (not deleted) this extension. I then had to re-enter my master password and synchronisation password (I self-host Joplin Server) and now it seems to work.
I also noticed you cannot have two versions of Joplin (intel and arm64) with different names in the Applications folder, if "Joplin 2" (arm64) was present, "Joplin" (intel) showed the same behavior described above (blank window, no error, can only force quit). Once "Joplin 2" was gone, it worked.

**Comment by iLucis** [comment]
> 

@yodahome Thanks! Just like you, I got a blank console too. I did a check based on your situation, but I still haven't found a solution. It's so strange.

@laurent22  
The same as he said, I opened the Development Tools, and all in it including the console was empty, with no errors. so I don't know how to confirm what is keeping Joplin from launching.

However, I checked my MacOS and there are not two versions of Joplin as he mentioned. My MacOS theme version is not v1.3.13 either, I also tried switching to other themes, but it had no effect. 

Hmm, Could it be that I need to roll back my MacOS system from Sonoma beta to the official Ventura?!

<img width="1210" alt="image" src="https://github.com/laurent22/joplin/assets/132722267/0b78b0b4-f580-42b2-8e0d-323d4103a179">
